src/frontends/pytorch/src/op/topk.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/topk.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/convert.hpp"
```

Comment:
@mvafin, @bszmelcz, I think also we need to include `opset` header as well here.

Reply:
Do not agree, operation headers are good here to speed up building